### ISSUE TITLE
Translations not being loaded

### ISSUE THREAD

**Issue by kagiura** [aralroca/next-translate#943]
**What version of this package are you using?** 1.6.0

**What operating system, Node.js, and npm version?** macOS 11.5.2, node v16.14.0, npm 8.19.2 (I use yarn 3.2.2 mainly). Oh, and I'm on Next 13, if that's related.

**What happened?**
I followed the Getting Started guide and couldn't seem to get it to work, the t() function just returns the key as-is. I even tried setting loadLocaleFrom to make it return null or even throw an error, but no erros are being reported and the app just runs normally. I'm not exactly sure why but it seems like the translations aren't even being loaded right now.

The repo is huge but it is available at https://github.com/enstars/makotools/tree/i18n. (I've done a lot of experimenting locally as well, but so far nothing's working. I also noticed all of the log-related configuration doesn't work either? I have a feeling it's something during initial load that's not working for some reason, as the library is dead silent in console even when I'm purposefully breaking it.)

**What did you expect to happen?**
Translations to load properly

**Are you willing to submit a pull request to fix this bug?**
Most likely not

Thank you so much!

**Comment by aralroca** [comment]
If you have `loadLocaleFrom` inside your `i18n.js` file the issue can be related with this one:

https://github.com/aralroca/next-translate/issues/851

This issue was introduced after the SWC rust compiler, this compiler doesn't support dynamic import in Node (files with module.export and require). 

As a workaround, I recomend this for now 
* https://github.com/aralroca/next-translate/issues/851#issuecomment-1173611946

This workaround is avoiding the `require`, which loads all the namespaces in Webpack instead the only one that you need.

**Comment by kamishirorui** [comment]
Hmm, I've had this issue even before I poked around with loadLocaleFrom, so I don't think it's related. (I did try the workaround you linked, but that also didn't work) This will probably break a few things but I'm gonna try downgrade to older versions of both next-tl and next inself in case it reveals anything.

**Comment by kamishirorui** [comment]
<img width="293" alt="image" src="https://user-images.githubusercontent.com/58155530/204095641-d06d3d64-a6cf-4629-b03c-9803838984c8.png">

I figured it out! Spent hours logging everything I could think of in the library, it turns out the HoC wasnt inserted (? honestly im not sure what the function does) in app.js in the first place because I used _app.page.tsx, not _app.tsx. I tried adding `.page` to the end of those comparisons and it worked like a charm. Seems like a feature to support [custom page extensions](https://nextjs.org/docs/api-reference/next.config.js/custom-page-extensions) would be needed here!
